google/cloud/bigtable/internal/table.h

```diff
@@ -118,6 +119,54 @@ class Table {
    */
   std::vector<FailedMutation> Apply(SingleRowMutation&& mut);
 
+  /**
+   * Make an asynchronous request to mutate a single row.
+   *
+   * @param mut the mutation to apply.
+   * @param cq the completion queue that will execute the asynchronous calls,
+   *     the application must ensure that one or more threads are blocked on
+   *     `cq.Run()`.
+   * @param callback a functor to be called when the operation completes. It
+   *     must satisfy (using C++17 types):
+   *     static_assert(std::is_invocable_v<
+   *         Functor, google::bigtable::v2::MutateRowResponse&,
+   *         grpc::Status const&>);
+   *
+   * @tparam Functor the type of the callback.
+   *
+   * @return A handle to the asynchronous operation, applications can request
+   *   the operation to be canceled.
+   */
+  template <typename Functor>
+  std::shared_ptr<AsyncOperation> AsyncApply(SingleRowMutation&& mut,
+                                             CompletionQueue& cq,
+                                             Functor&& callback) {
+    using AsyncOpResult =
+        AsyncUnaryRpcResult<google::bigtable::v2::MutateRowResponse>;
+
+    auto rpc_policy = rpc_retry_policy_->clone();
```

Comment:
Yes. The retry policy keeps track of the number of retries, or the time elapsed since you started trying. The backoff policy changes and increases the sleep time on each retry.